### PR TITLE
fix(learn): progress bar label alignment

### LINKED ISSUE #48909: Percent label next to progress bar is not vertically aligned.
### Describe the Issue

In the new projects, the `1%` in the image below is not vertically aligned with the progress bar next to it.

Screen Shot 2023-01-03 at 10 00 30 AM

### Steps to Reproduce

1. Go to https://www.freecodecamp.org/learn/2022/responsive-web-design
2. Click on one of the blocks and complete a challenge
3. Go back to https://www.freecodecamp.org/learn/2022/responsive-web-design
4. Find the block where you completed that challenge and make sure it's collapsed to see the progress bar

### Expected behavior

The progress bar and percent label next to it to be vertically aligned

### PR DESCRIPTION
**Pull request by sleptsov**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #48909

This is a simple fix for the alignment of the progress bar value label. The solution proposed by maintainers during issue discussion.

### CODE REVIEW

**Comment by Sboonny** [comment]
that solution is outdated, in the latest comment, we have decided to go with "Lato" font

**Comment by sleptsov** [comment]
> that solution is outdated, in the latest comment, we have decided to go with "Lato" font

You're right, it was my mistake, switched from margin to font change.

**Review by Sboonny — COMMENTED** [review]
unfortunately, it isn't that simple, luckily there is a PR similar to this https://github.com/freeCodeCamp/freeCodeCamp/pull/47767/files

You can apply similar changes to the CSS files, without changing the title or code.

@ahmadabdolsaheb, just wondering do you mean the headings elements by title correct?

```css
h1,
h2,
h3,
h4,
h5,
h6
```

File: client/src/templates/Introduction/intro.css [hunk 1]
```diff
@@ -459,6 +459,7 @@ ToDo: find out why, and remove the need for it */
 
 .block-grid .progress-wrapper span {
   color: var(--quaternary-color);
+  font-family: 'Lato', monospace;
```
[the remark this answers is not in the document]
Comment (sleptsov) [reply client/src/templates/Introduction/intro.css]: Challenge accepted )))

[the remark this answers is not in the document]
Comment (sleptsov) [reply client/src/templates/Introduction/intro.css]: 1. Font declaration moved to variables.
2. `Lato` is used as the primary font.
3. `Hack-ZeroSlash` is used as secondary and explicitly for `h1-h6` and `code` tags.
4. Explicit font declaration in styles removed as `Lato` is default now, except where `Lato` was set as a font in the landing page and donation page.
5. `Sax Mono` font was removed as its source is missing in a codebase.
6. For the particular cases with progress value label alignment, we need to set font-family directly for `.progress-wrapper` as it is wrapped in the `h3` tag and uses 'Hack-ZeroSlash'.

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @sleptsov, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Review by moT01 — COMMENTED** [review]
I did a side-by-side of this PR and production. There's quite a few differences in the fonts in places - see images below:


<details>
<summary>Images</summary>
This PR is on the left in all images, production is on the right in all images.
<br /><br />

Landing page:
Screen Shot 2023-03-13 at 12 58 16 PM

Screen Shot 2023-03-13 at 12 58 40 PM

/learn:
Screen Shot 2023-03-13 at 1 00 16 PM

Superblock page:
Screen Shot 2023-03-13 at 1 00 42 PM

Screen Shot 2023-03-13 at 1 04 43 PM

Challenge:
Screen Shot 2023-03-13 at 1 02 45 PM

Screen Shot 2023-03-13 at 1 05 56 PM

Completion Modal:
Screen Shot 2023-03-13 at 1 06 29 PM

/settings:
Screen Shot 2023-03-13 at 1 18 40 PM

Screen Shot 2023-03-13 at 1 19 33 PM

Profile:
Screen Shot 2023-03-13 at 1 22 21 PM

Certification:
Screen Shot 2023-03-13 at 1 21 25 PM
</details>

This shows all the main changes in fonts on the site. If something isn't shown above, it probably looked almost the same on this PR and production. Can you confirm that this is what we want @ahmadabdolsaheb. If this is fine, it LGTM 🎉

**Comment by ahmadabdolsaheb** [comment]
The text is more readable. Thanks for the clean up.
A few notes:
- Lato is a sans serif font, so the back up should also be a sans serif: --font-family-primary: 'Lato',  sans-serif;
- To make the variable naming more descriptive, the names can be changed to --font-family-sans-serif, --font-family-monospace
Other than that: LGMT

**Comment by sleptsov** [comment]
> The text is more readable. Thanks for the clean up. A few notes:
> 
> * Lato is a sans serif font, so the back up should also be a sans serif: --font-family-primary: 'Lato',  sans-serif;
> * To make the variable naming more descriptive, the names can be changed to --font-family-sans-serif, --font-family-monospace
>   Other than that: LGMT

Thanks for reviewing! Font fallback and variable names changed according to your comments.

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**